src/main/validation/PromotionQueryValidation.ts

```diff
@@ -45,5 +49,8 @@ export interface PromotionQueryDTO {
   cuisine?: CuisineType | CuisineType[];
   expirationDate?: Date;
   dayOfWeek?: Day;
+  sort?: SortOptions;
+  lat?: number;
+  lon?: number;
```

Comment:
Kind of a challenge if you want, totally okay if you don't want to:

How about we add some validation such that if the sort option is Distance, then we need to make sure lat/lon are defined